libpd_wrapper/s_libpdmidi.c

```diff
@@ -52,7 +53,9 @@ void outmidi_byte(int port, int value) {
 }
 
 // The rest is not relevant to libpd.
```

Comment:
I suppose this line needs to move down now that sys_get_midi_apis does something.